src/main.rs

```diff
@@ -130,23 +136,31 @@ struct Toolchain<'a> {
     host_target: &'a str,
     rust_std_targets: &'a [&'a str],
     components: &'a [&'a str],
+    name: &'a str,
 }
 
 fn install_single_toolchain(
     client: Option<&Client>,
     prefix: &str,
     toolchains_path: &Path,
     toolchain: &Toolchain,
+    force: bool
 ) -> Result<(), Error> {
     let dest = if toolchain.alt {
-        Cow::Owned(format!("{}-alt", toolchain.commit))
+        Cow::Owned(format!("{}-alt", toolchain.name))
     } else {
-        Cow::Borrowed(toolchain.commit)
+        Cow::Borrowed(toolchain.name)
```

Comment:
If we run `--alt --name somename` we'll end up with a `somename-alt` toolchain, which might be surprising.

Perhaps move this logic to where the struct is constructed?